### ISSUE TITLE
HorizontalPageBreakRepeat column page break can continue on same page  if verical space is enough

### ISSUE THREAD

**Issue by damosvag-ali** [simonbengtsson/jsPDF-AutoTable#1051]
HorizontalPageBreakRepeat column page break can continue on same page  if vertical space is enough to finish not create a new page

**Comment by mmghv** [comment]
What do you mean by that? can you give an example?

**Comment by damosvag-ali** [comment]
Hi Thank you for your response 
As you see on the Pdf Created there is enough space to continue printing on the first page
[document.pdf](https://github.com/simonbengtsson/jsPDF-AutoTable/files/15105319/document.pdf)

**Comment by mmghv** [comment]
You mean in case it's a single page and there's vertical space to continue printing the second horizontal page in the current page?

That sounds like a good idea. maybe could be the default behavior.

**Comment by damosvag-ali** [comment]
Yes , It is very nice option i have it on Desktop application using Delphi
with Fastreport CrossTab Object printing large Horizontal columns,
if you make the update can inform me

Thank You

**Comment by mmghv** [comment]
This could take a while since we're currently working on v4, but I'll keep it in my mind.

**Comment by solveigsun** [comment]
We are also using your plugin and would love this feature!

**Comment by pooya1361** [comment]
Hi, I'm facing the same issue. Is this resolved somehow? Please guide me if there's a solution
